api/graphql/makeModels.js

```diff
@@ -105,6 +109,27 @@ export default function makeModels (userId, isAdmin) {
         q.where('comments.post_id', 'in', PostMembership.query().select('post_id')
           .where('community_id', 'in', myCommunityIds()))
       })
+    },
+
+    link_previews: {
+      typename: 'LinkPreview',
+      model: LinkPreview,
+      attributes: [
+        'id',
+        'title',
+        'url',
+        'image_url'
+      ],
+      getters: {
+        title: c => c.get('title'),
+        url: c => c.get('url'),
+        imageUrl: c => c.get('image_url')
+      }
+      // TODO: filter for linkPreviews
```

Comment:
as long as we never have an API that allows you to look up link previews by ID directly, then we wouldn't need a filter for them. because we'd only get them via their related post, and we'd already be doing filtering/policies to restrict your access to posts.